packs/git/actions/clone.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: clone
+runner_type: run-remote
+description: "Clone a repository"
+enabled: true
+entry_point: ''
+parameters:
+  source:
```

Comment:
Nope, I'm using it for a private repository. It just requires that SSH Keys exist on the system. This is not how I envision the end-state to be, but it's Good Enough(tm) for now.

I would like to be able to add flags like AUTH or SSH_KEY without having to write a secondary script to handle the parameter passing and assembling the final command.